hazelcast/src/main/java/com/hazelcast/monitor/LocalGCStats.java

```diff
@@ -18,6 +18,9 @@
 
 import com.hazelcast.memory.GarbageCollectorStats;
 
+/**
+ * Interface for local GC statistics to be used by {@link MemberState}
```

Comment:
Normally you don't state that something is an interface.

Reply:
Fixed